src/js/models/menubar.js

```diff
@@ -227,6 +227,20 @@ define(function (require, exports, module) {
                 (document.layers.selectedNormalized.size === 1) &&
                 (document.layers.selected.every(function (layer) {
                     return layer.vectorMaskEnabled === true;
+                })),
```

Comment:
These two rules should be just testing for hasVectorMask/canAddVectorMask, and you should add `layers-selected-1` as the other rule.